xray/Cargo.toml

```diff
@@ -23,7 +23,7 @@ nalgebra = "0.22.0"
 num = "0.3.0"
 pbr = "1.0.3"
 protobuf = "2.18.0"
-rayon = "1.4.1"
+rayon = "1.5.1"
```

Comment:
I recommend to update the rayon version in all `Cargo.toml` files, so we are consistent throughout the repository

Reply:
Done. Interesting, this time's change does not generate new changes in cargo.lock